streaming_providers/pikpak/utils.py

```
@@ -157,39 +158,41 @@ async def initialize_pikpak(user_data: UserData):
+            token_refresh_callback=store_pikpak_token_in_cache,
+            token_refresh_callback_kwargs={"user_data": user_data},
+        )
 
-    try:
-        await pikpak.login()
-    except PikpakException as error:
-        if "Invalid username or password" == str(error):
+        try:
+            await pikpak.login()
+        except PikpakException as error:
+            if "Invalid username or password" == str(error):
```

Ruff

Yoda condition detected

Rewrite as `str(error) == "Invalid username or password"`

(SIM300)